lua/astrocommunity/editing-support/copilotchat-nvim/init.lua

```diff
@@ -71,9 +70,14 @@ return {
         -- Helper function to create mappings
         local function create_mapping(action_type, selection_type)
           return function()
-            require("CopilotChat.integrations.telescope").pick(require("CopilotChat.actions")[action_type] {
-              selection = require("CopilotChat.select")[selection_type],
-            })
+            local fzf_ok = pcall(require, "fzf-lua")
+            local snacks_ok = pcall(require, "snacks")
+
+            require("CopilotChat.integrations." .. (fzf_ok and "fzflua" or snacks_ok and "snacks" or "telescope")).pick(
```

Comment:
Is this valid? I dont think it supports snacks

Reply:
https://github.com/CopilotC-Nvim/CopilotChat.nvim/blob/main/lua/CopilotChat/integrations/snacks.lua

Brand new I guess

Reply:
Lmao not in the docs.